Star Trek Adventurers 2d20/sheet.json

```diff
@@ -0,0 +1,7 @@
+{
+	"html": "Star Trek Adventures 2d20.html",
+	"css": "Star Trek Adventures 2d20.css",
+	"authors": "David Collier",
+	"preview": "Star Trek Adventures 2d20.PNG",
```

Comment:
Like this: https://github.com/Roll20/roll20-character-sheets/blob/master/Adventures%20in%20Middle%20Earth/sheet.json#L5